### ISSUE TITLE
Wetland output file trouble

### ISSUE THREAD

**Issue by hinckleyb** [chrisschuerz/SWATplusR#64]
Hey, I am running the below code to try and get water balance outputs from the model. I have been able to get data from most sources but run into trouble with "wetland". Is this output defined in SWATplusR 0.6.4?

Thanks,
Brian

q_simn <- run_swatplus(project_path = path, 
                       output = list(flow = define_output(file = 'channel_sd',
                                                          variable = 'flo_out',
                                                          unit = 44),
                                     no3 = define_output(file = "channel_sd",
                                                         variable = "no3_out",
                                                         unit = 44),
                                     GW = define_output(file = "basin_aqu",    #Basin-wide aqu flow into channels (mm H2O)
                                                         variable = "flo_cha",
                                                         unit = 1),
                                     no3_aquload = define_output(file = "basin_aqu",
                                                                  variable = "no3gw", #basin wide Nitrate loading to reach from aqu (kg N/ha)
                                                                  unit = 1),
                                     no3_aquloss = define_output(file = "basin_aqu",
                                                                     variable = "nloss", #basin wide NO3 loss (kg/ha)
                                                                     unit = 1),
                                     crop_yield = define_output(file = "basin_pw",
                                                                 variable = "yield", #basin-wide Crop yield (kg/ha)
                                                                 unit = 1),
                                     RunoffNO3loss = define_output(file = "basin_ls", #basin wide NO3 transported in runoff (kg N/ha)
                                                                 variable = "surqno3",
                                                                 unit = 1),
                                     lateralNO3loss = define_output(file = "basin_ls", #basin wide NO3 transported in lateral flow (kg N/ha)
                                                                 variable = "lat3no3",
                                                                   unit = 1),
                                     Fert = define_output(file = "basin_nb",
                                                                    variable = "fertn",   #basin wide nitrogen fertilizer applied (kg N/ha)
                                                                    unit = 1),
                                     chan_runoff = define_output(file = "basin_wb", #basin wide runoff flowing into channels (mm H2O)
                                                          variable = "surq_cha",
                                                          unit = 1),
                                     latq = define_output(file = "basin_wb",
                                                            variable = "latq_cha", #basin wide lateral flow into channels (mm H2O)
                                                            unit = 1),
                                     overbank = define_output(file = "basin_wb", #basin wide overbank flooding from channels (mm H2O)
                                                          variable = "overbank",
                                                          unit = 1),
                                     Chinqchan_runoff = define_output(file = "hru_wb", #basin wide runoff flowing into channels (mm H2O)
                                                                 variable = "surq_cha",
                                                                 unit = 660),
                                     Chinqlatq = define_output(file = "hru_wb",
                                                          variable = "latq_cha", #basin wide lateral flow into channels (mm H2O)
                                                          unit = 660),
                                     ChinqGW = define_output(file = "aquifer",    #Basin-wide aqu flow into channels (mm H2O)
                                                        variable = "flo_cha",
                                                        unit = 31),
                                     ChinqWet = define_output(file = "wetland",    #Basin-wide aqu flow into channels (mm H2O)
                                                             variable = "flo_out",
                                                             unit = 660)
                                     ),
                       
                       parameter = par_set,
                       n_thread = 10)

**Comment by chrisschuerz** [comment]
Also sorry for the very late reply here. Wetland outputs are not defined in the print.prt (at least in setups that I have) and can therefore not be returned to R. Only the object files which are defined in print.prt can be adressed (in the upcomung version of `SWATrunR` 1.0.0, as the package is named now) you can also get mgtout and fdcout outputs but still no wetlands.
